Portal/src/Datahub.Application/Configuration/DatahubPortalConfiguration.cs

```diff
@@ -257,6 +258,14 @@ public bool IsFeatureEnabled
     }
 }
 
+public class StorageConfiguration
+{
+    public string BlockedFileExtensions { get; set; } = ".exe,.dll,.bat,.cmd,.sh,.js,.jar,.msi,.com,.scr,.pif,.cpl";
```

Comment:
Per [Line's recommendation](https://163gc.sharepoint.com/:w:/r/sites/ScienceProgram/Shared%20Documents/Data%20Solutions%20for%20Science/Federal%20Science%20DataHub%20(FSDH)/Project%20Planning%20%26%20Execution/FY%2025-26%20Architecture%20High%20Level%20Approach/Security%20Requirements%20for%20Antivirus.docx?d=w18a58d2c0e82425ab3c999476199ed5d&csf=1&web=1&e=3WJYO5), can we block all of the following:

```suggestion
    public string BlockedFileExtensions { get; set; } = ".ace,.ade,.adp,.ani,.app,.apk,.bas,.bat,.chm,.cmd,.com,.cpl,.crt,.docm,.dll,.exe,.hlp,.ht,.hta,.inf,.ins,.isp,.jar,.job,.js,.jse,.lnk,.mda,.mdb,.mde,.mdz,.msc,.msi,.msp,.mst,.pcd,.pif,.reg,.scr,.sct,.shs,.url,.vb,.vbe,.vbs,.wsc,.wsf,.wsh";
```